apps/ledger-live-desktop/src/renderer/reducers/wallet.ts

```diff
@@ -38,6 +38,15 @@ export const useMaybeAccountName = (
     !account ? undefined : accountNameWithDefaultSelector(state.wallet, account),
   );
 };
+export const useBatchMaybeAccountName = (
+  accounts: (AccountLike | null | undefined)[],
+): (string | undefined)[] => {
+  return useSelector((state: State) =>
+    accounts.map(account =>
```

Comment:
Can we put the content of this map in a function to factorize with the `useMaybeAccountName` function above